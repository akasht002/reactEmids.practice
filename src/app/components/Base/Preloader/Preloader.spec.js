import React from 'react';
import {Preloader} from './index';
import renderer from 'react-test-renderer';

 test('Preloader Component Testing', () => {
  const component = renderer.create(
    <Preloader className="Loader"></Preloader>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});