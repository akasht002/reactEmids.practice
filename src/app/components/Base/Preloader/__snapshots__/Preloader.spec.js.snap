// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader Component Testing 1`] = `
<div
  className="Loader"
/>
`;
